tests/wpt/grouping_formatter.py

```diff
@@ -47,16 +49,25 @@ def __init__(self):
             'CRASH': [],
         }
 
+    def get_move_up_and_clear_eol_codes(self):
+        try:
+            import blessings
+        except ImportError:
+            return DEFAULT_MOVE_UP_CODE, DEFAULT_CLEAR_EOL_CODE
+
+        try:
+            self.terminal = blessings.Terminal()
+            return self.terminal.clear_eol, self.terminal.move_up
+        except Exception as exception:
+            sys.stderr.write("GroupingFormatter: Could not get terminal "
+                             "control characters: %s\n" % exception)
+            return DEFAULT_MOVE_UP_CODE, DEFAULT_CLEAR_EOL_CODE
+
     def text_to_erase_display(self):
         if not self.interactive or not self.current_display:
             return ""
-
-        # TODO(mrobinson, 8313): We need to add support for Windows terminals here.
-        erase_length = len(self.current_display)
-        if self.one_line:
-            return "\b \b" * erase_length
-        else:
-            return ("\033[F" + "\033[K") * len(self.current_display.splitlines())
+        return ((self.move_up + self.clear_eol) *
+                len(self.current_display.splitlines()))
```

Comment:
I would expect splitting a constant string on a constant separator to work the same on all platforms :) The real question is given multiline string content from an external source, does it contain `\r` that would cause difficulties when splitting on `\n`?